packages/css/src/components/link/link.scss

```diff
@@ -68,8 +60,4 @@
   &:hover {
     color: var(--ams-link-inverse-color-hover-color);
   }
-
-  &:visited {
```

Comment:
Does this work? If we depend on the browser default color for visited links and you have it on a darker background the purple might provide a very poor contrast.

Reply:
This PR does not introduce a regression for the colour of visited links on a coloured background. You can add a link to the content of a spotlight on the production Storybook to verify. A story to implement link colours is on our next sprint.